src/Settings/settings.json

```diff
@@ -1239,5 +1239,11 @@
         "name": "Promotion Dialog Shown",
         "type": "bool",
         "notr": true
+    },
+    {
+        "name": "Auto Uncheck Accepted Testcases",
+        "desc": "Auto uncheck accepted testcases",
+        "type": "bool",
+        "default": "false"
```

Comment:
Formatting issue:

You can make sure it's correctly formatted it in two ways:

1. Copy paste an already formatted setting and edit it (Recommended)
2. Format using vscode default JSON formatter.